build.gradle

```diff
@@ -48,20 +48,14 @@ repositories {
 
 dependencies {
     compile gradleApi()
-    compile 'com.android.tools.build:gradle:1.5.0'
-    compile 'com.google.code.gson:gson:2.5'
+    compile 'com.android.tools.build:gradle:2.1.0'
```

Comment:
OK, I ran a test with the plugin using 2.1.0 and the app using 1.5.0; everything worked.